src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.search-bar--big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input:focus {
+  outline: none;
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
this property overwrites background-image property, so you should add it here as well:

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/dc376cf0-5e46-4c40-b985-ca1b67041c15)

Here are some related links:
https://www.geeksforgeeks.org/css-combine-background-image-with-gradient-overlay/

https://stackoverflow.com/questions/2504071/how-do-i-combine-a-background-image-and-css3-gradient-on-the-same-element